package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "mc-components",
   "version": "0.2.0",
-  "main": "./dist/components/index.js",
+  "main": "./dist/index.js",
```

Comment:
The file `./dist/index.js` doesn't exist

Reply:
Good catch, I'll upload the dist files too?